### DISCUSSION TITLE
Is there a way to assign a particular table index to a function?

### DISCUSSION THREAD

**Discussion by debevv** [emscripten-core/emscripten#19426] [Q&A] [answered]
Hi, I was wondering if there is a way, with some kind of pragma maybe, to tell the compiler to assign a specific table index to a function, for example:
```c
void func(int a, int b) {
  /* body */ 
}
#pragma index (func, 5)
```

**Comment by sbc100** [comment, marked as answer]
No, there is no way to do this.  The linker (wasm-ld) assigns indexes to functions at link time and there no way to control what the index will be.
